src/main/java/Duke.java

```diff
@@ -1,10 +1,210 @@
+
+            // Updating the command word, catching any invalid inputs
+            try {
+                command = CommandWord.getCommand(commandWord);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+
+            // Print out goodbye message and exits the program
+            if (command == CommandWord.BYE) {
+                exitJukebox();
+                break;
+            }
+
+            // Prints out the task list
+            if (command == CommandWord.LIST) {
+                printList(taskList);
+                continue;
+            }
+
+            // Marks or unmarks the given task
+            if (command == CommandWord.MARK || command == CommandWord.UNMARK) {
+                markUnmarkTask(command, inputWordsSplit);
+                continue;
+            }
+
+            // Add a new task based on the given keyword
+            if (command == CommandWord.TODO || command == CommandWord.DEADLINE || command == CommandWord.EVENT) {
+                try {
+                    addNewTask(command, inputWordsSplit[1]);
+                    continue;
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("Hmm... looks like you don't have a description for this task :( Please try again!\n");
+                }
+            }
+
+            // Deletes a task from the task list
+            if (command == CommandWord.DELETE) {
+                deleteTask(inputWordsSplit);
+            }
+        }
+    }
+
+
+        /**
+         * Prints out the task list iteratively.
+         * @param taskList Input task list
+         */
+        private static void printList (ArrayList < Task > taskList) {
+            if (!taskList.isEmpty()) {
+                System.out.println("These are your current tasks! :)");
+                int i = 1;
+                for (Task t : taskList) {
+                    String s = String.format("%d.%s", i, t);
+                    System.out.println(s);
+                    i++;
+                    if (t == null) {
+                        break;
+                    }
+                }
+                System.out.println(" ");
+            } else {
+                System.out.println("No tasks have been added yet!\n");
+            }
+        }
+
+        /**
+         * Prints the current number of tasks in the task list.
+         * @param taskList Input task list
+         */
+        private static void printCurrentTasks (ArrayList <Task> taskList) {
+            System.out.println(String.format("And... that makes %d task(s) in your list! :)", taskList.size()));
+        }
+
+        /**
+         * Marks or unmarks the task.
+         * @param command Input command word
+         * @param inputWordsSplit String array of the command word with the task number
+         */
+        private static void markUnmarkTask (CommandWord command, String[] inputWordsSplit){
```

Comment:
I think there should be a whitespace before `{`. Also, can the method be better named (eg. `unmarkTask`)?